# Request
geometry_msgs/Pose target_pose
geometry_msgs/Pose charging_station_pose
---
# Response
bool success
string message
